src/main/java/com/internet/shop/dao/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,178 @@
+package com.internet.shop.dao;
+
+import com.internet.shop.exceptions.DataBaseProcessingException;
+import com.internet.shop.lib.Dao;
+import com.internet.shop.models.Order;
+import com.internet.shop.models.Product;
+import com.internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class OrderDaoJdbcImpl implements OrderDao {
+    @Override
+    public List<Order> getUserOrders(Long userId) {
+        List<Order> orders = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            String query = "SELECT * FROM orders WHERE deleted = FALSE AND user_id=?";
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setLong(1, userId);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                Order order = new Order();
+                order.setId(resultSet.getLong("order_id"));
+                order.setIdUser(resultSet.getLong("user_id"));
+                orders.add(order);
+            }
+        } catch (SQLException e) {
+            throw new DataBaseProcessingException("Order list was not created", e);
```

Comment:
```suggestion
            throw new DataProcessingException("Order list was not created", e);
```

Reply:
We're processing some information - data 
that we receive from database or database receives from our app
Let's fix the naming

Reply:
rename DataProcessingException